app/_locales/it/messages.json

```diff
@@ -799,9 +845,35 @@
   "invalidBlockExplorerURL": {
     "message": "URI Block Explorer invalido"
   },
+  "invalidCustomNetworkAlertContent1": {
+    "message": "Il blockchain ID per la rete personalizzata '$1' deve essere reinserito.",
+    "description": "$1 is the name/identifier of the network."
+  },
+  "invalidCustomNetworkAlertContent2": {
+    "message": "Per proteggerti da fornitori di rete maligni o non funzionanti, i blockchain ID sono ora richiesti per tutte le reti personalizzate."
```

Comment:
same as above, should "blockchain ID" be kept "chain ID" since there isn't a known concept called "blackchain ID?" Alternatively I think it could be translated "ID catena"